x-pack/test_serverless/functional/test_suites/common/discover/context_awareness/index.ts

```diff
@@ -25,7 +25,6 @@ export default function ({ getService, getPageObjects, loadTestFile }: FtrProvid
       await kibanaServer.uiSettings.update({
         'timepicker:timeDefaults': `{ "from": "${from}", "to": "${to}"}`,
       });
-      await PageObjects.svlCommonPage.login();
```

Comment:
If we remove it from the `index`, should not we add login to all loaded files?

```
loadTestFile(require.resolve('./_root_profile'));
loadTestFile(require.resolve('./_data_source_profile'));
loadTestFile(require.resolve('./extensions/_get_row_indicator_provider'));
loadTestFile(require.resolve('./extensions/_get_doc_viewer'));
```

I can see that it's already added to `loadTestFile(require.resolve('./_data_source_profile'));`. Do you know why CI is not reporting that it's not added to other files yet?

Reply:
You mean throwing an error?

Reply:
Yeah, what user does it use for the remaining tests then?

Reply:
Well to answer somewhat round-about, the additional purpose of this pr is to ask you all responsible kibana dev, what roles exactly should be used.  I, as appex-qa dev, can't tell you.  

My argument will always be the least privileged role.  

As to why they are not throwing errors, my guess is that many if not all were originally developed to use the operator user.  The operator user may as well be root, or superuser.  Which is not how an actual user runs them.

When this pr is merged, they all should be running just like the actual users in the ui.

Reply:
If the idea is to require a login in each test suit, I would expect that tests then running in isolation. If login is not called then there should be no active session, which seems to be not the case right now. I assume that it reuses the login session from a previous test suite. Very unpredictable in my opinion.

Having an explicit login in `index` file seems to be preferable at this point then. It least it's explicit what permissions are set for the loaded suites.

Reply:
I concur, 100%

Reply:
Which role @jughosta ?

Reply:
`viewer` might be sufficient, thanks!

Reply:
for these files correct?

loadTestFile(require.resolve('./_root_profile'));
loadTestFile(require.resolve('./_data_source_profile'));
loadTestFile(require.resolve('./extensions/_get_row_indicator_provider'));
loadTestFile(require.resolve('./extensions/_get_doc_viewer'));

Reply:
[b1796c8](https://github.com/elastic/kibana/pull/188128/commits/b1796c84f766df4bc0274265d5a28f47bc15a975)

Reply:
@jughosta may I have one more review, pretty please? 😄